lib/auth.js

```diff
@@ -417,22 +256,24 @@ api.createIdentity = function(actor, identity, callback) {
 };
 
 /**
- * Gets the Identity ID(s) that match the given email address.
+ * Gets an IdentityID from the URL parameters in the given request. In the
+ * common case, the authenticated user for the request can be used to obtain
+ * the ID without an extra database hit.
  *
- * @param email the email address.
- * @param callback(err, identityIds) called once the operation completes.
+ * @param req the request.
+ * @param callback(err, identityId) called once the operation completes.
  */
-api.resolveEmail = function(email, callback) {
```

Comment:
To clarify, if these functions are never exposed outside of `bedrock-idp` we don't need to keep them available as deprecated. However, if they are -- we'll want to see if we can easily preserve them.